index.html

```diff
@@ -226,7 +250,7 @@ <h4>Skills</h4>
       </div>
     </section>
 
-    <footer class="flex footer">
+    <footer id="contact" class="flex footer">
       <div class="flex footer-wrapper">
         <h2 class="section-title">Contact me</h2>
         <p class="section-description">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
```

Comment:
- [ ] The width of the form fields is a little bigger than the design. Please reduce the form fields width and make sure it matches with the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/165697881-a09b695e-249f-4f33-8b82-66333b5901a8.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/165697930-26194b2a-1281-4f63-b7c3-cb85787b9b17.png)